devtools/cli/src/main/java/io/quarkus/cli/Remove.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.cli;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
+import io.quarkus.devtools.commands.RemoveExtensions;
+import io.quarkus.devtools.commands.data.QuarkusCommandOutcome;
+import io.quarkus.devtools.project.QuarkusProject;
+import io.quarkus.platform.descriptor.QuarkusPlatformDescriptor;
+import io.quarkus.platform.tools.config.QuarkusPlatformConfig;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "remove", aliases = "rm", mixinStandardHelpOptions = false, description = "Remove an extension from this project.")
+public class Remove implements Callable<Integer> {
+
+    @CommandLine.Option(names = { "-h", "--help" }, usageHelp = true, description = "display this help message")
+    boolean usageHelpRequested;
+
+    @CommandLine.Parameters(arity = "1", paramLabel = "EXTENSION", description = "extension to remove")
+    Set<String> extensions;
+
+    @CommandLine.ParentCommand
+    QuarkusCli parent;
+
+    @CommandLine.Spec
+    CommandLine.Model.CommandSpec spec;
+
+    @Override
+    public Integer call() {
+        final QuarkusPlatformDescriptor platformDescr = QuarkusPlatformConfig.getGlobalDefault().getPlatformDescriptor();
+        try {
+            Path projectDirectory = Paths.get(System.getProperty("user.dir"));
+            RemoveExtensions project = new RemoveExtensions(QuarkusProject.resolveExistingProject(projectDirectory,
+                    platformDescr)).extensions(extensions);
+            QuarkusCommandOutcome result = project.execute();
+            return result.isSuccess() ? CommandLine.ExitCode.OK : CommandLine.ExitCode.SOFTWARE;
+        } catch (Exception e) {
+            if (parent.showErrors)
+                e.printStackTrace(spec.commandLine().getErr());
+            spec.commandLine().getErr().println("Unable to remove extension matching:" + e.getMessage());
```

Comment:
`quarkus rm hibernate` in a project with just `quarkus-hibernate-orm-panache` still complains about having to be precise:

```
❌ Multiple extensions matching 'hibernate'
     * io.quarkus:quarkus-hibernate-orm
     * io.quarkus:quarkus-hibernate-reactive
     * io.quarkus:quarkus-hibernate-envers
     * io.quarkus:quarkus-hibernate-orm-panache
     * io.quarkus:quarkus-hibernate-orm-rest-data-panache
     * io.quarkus:quarkus-hibernate-orm-panache-kotlin
     * io.quarkus:quarkus-hibernate-search-elasticsearch
     * io.quarkus:quarkus-hibernate-validator
     Be more specific e.g using the exact name or the full GAV.
```

Reply:
This should be logged as a separate issue.  Not a CLI thing.